### PR TITLE
#162701212 Paginate the floors response

### PR DESCRIPTION
**Pull request by kampkelly**
#### What does this PR do?
- Paginate the response coming from the floors

#### Description of Task to be completed?
The task is to allow the admin to be able to scroll through all the available floors .

#### How should this be manually tested?
- Run the server `http://127.0.0.1:5000/mrm`.
- Run the following query
```
query {
  allFloors(page: 1, perPage: 4) {
    floors {
      name
       rooms {
        name
      }
    }
     queryTotal
    hasNext
    hasPrevious
  }
}
```

**Any background context you want to provide?**
None

**What are the relevant pivotal tracker stories?**
[#162701212](https://www.pivotaltracker.com/story/show/162701212)

#### Screenshots
screen shot 2018-12-21 at 10 12 53 am

**Questions:**
None

### CODE REVIEW

File: alembic/versions/182e16720139_migration_message.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+"""Migration message
+
```
[the remark this answers is not in the document]
Comment (kampkelly) [reply alembic/versions/182e16720139_migration_message.py]: Okay, thanks. It has been done.

**Comment by Bonifase** [comment]
LGTM. However, use a brief description in your commit message especially the first line. You may want to have a look at the Andela engineering-playbook guide.

**Comment by kampkelly** [comment]
@Bonifase thanks, the commit message has been modified.

**Comment by joshuaocero** [comment]
@kampkelly please fix failing tests

**Comment by kampkelly** [comment]
@joshuaocero the tests are passing now

**Pull request merged**